docs/redirectless.md

```diff
@@ -33,7 +33,7 @@ authnWidget.initRedirectless(config);
 ## Configuration object
 There are multiple options for creating the configuration object to initiate the redirectless flow:
```

Comment:
It's a bit confusing the way the device authorization case is added here as number 2 in the list of options. It's not really another option for how to initiate redirectless, it's a different flow type.

Reply:
I'll make some changes.